src/containers/TransferForm/Form/Info.js

```diff
@@ -16,8 +17,13 @@ function Info({ onSubmit }) {
   const { symbol } = useParams()
   const token = useToken(symbol)
 
+  const web3 = Web3Service.getInstance()
+
+  const addressValidate = (value) =>
+    !web3.utils.isAddress(value) ? 'Invalid address' : true
```

Comment:
I'm thinking here, since we just use web3 instance inside this function, it is better to declare it inside addressValidate, otherwie it will be re-created for every re-render 🤔

Reply:
was resolved.